api/specs/web-server/_projects_nodes.py

```diff
@@ -44,36 +46,34 @@
     response_model=Envelope[NodeCreated],
     status_code=status.HTTP_201_CREATED,
 )
-def create_node(project_id: str, body: NodeCreate):
+def create_node(project_id: str, body: NodeCreate):  # noqa: ARG001
     ...
 
 
 @router.get(
     "/projects/{project_id}/nodes/{node_id}",
-    response_model=Envelope[NodeGet | NodeGetIdle],
-    # responses={"idle": {"model": NodeGetIdle}}, TODO: check this variant
+    response_model=Envelope[NodeGetIdle | NodeGetUnknown | DynamicServiceGet | NodeGet],
```

Comment:
I find very peculiar, that you need to change the API interface so much here. but we already discussed that.

Reply:
I have just updated the interface (added a missing type and a new one).
I have created an issue to keep track of this endpoint's problems.